resources/assets/components/Users/UserInformation.js

```diff
@@ -14,7 +14,20 @@ const UserInformation = props => (
         }
         <p>
           {props.user.email ? <span>{props.user.email}<br /></span> : null}
-          {props.user.mobile ? <span>{props.user.mobile}<br /></span> : null }
+
+          {
+            (() => {
+              if (props.user.mobile) {
+                if (props.linkSignup && isEmpty(props.user.first_name)) {
+                  return (<span><a href={`/signups/${props.linkSignup}`}>{props.user.mobile}</a><br /></span>);
+                } else {
+                  return (<span>{props.user.mobile}<br /></span>);
+                }
+              }
+              return (null);
+            })()
```

Comment:
These `()` look funny can you explain them to me?

Reply:
@katiecrane great q! I was trying to find a way to do an `if else` statement within JSX and found [this](https://stackoverflow.com/questions/45429836/immediately-invoked-function-expression-inside-jsx) on Stack overflow, which is where I learned how to do this!

Reply:
@chloealee a-ha! thanks!